packages/nimble-components/src/table-column/icon/cell-view/index.ts

```diff
@@ -77,8 +79,17 @@ export class TableColumnIconCellView
             this.text = mappingConfig.text;
             this.visualizationTemplate = mappingConfig.spinnerTemplate;
             this.textHidden = mappingConfig.textHidden;
+        } else if (mappingConfig instanceof MappingTextConfig) {
+            this.text = mappingConfig.text;
+            this.textHidden = false;
         }
     }
+
+    private resetState(): void {
+        this.text = undefined;
```

Comment:
wanna reset the severity state too? just all the state pulled from the mapping config